kitt_bar_app/assets/batches.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
I think this file should be gitignored, but created empty (if it doesn't exist) on some sort of setup function call that our potential hundreds of millions of user would have to call on first installation.

What do you think? 😬